ios/FluentUI.Demo/FluentUI.Demo/DemoAppearanceView.swift

```diff
@@ -0,0 +1,156 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import SwiftUI
+
+struct DemoAppearanceView: View {
+
+    @Environment(\.colorScheme) var systemColorScheme: ColorScheme
+
+    @ObservedObject var configuration: Configuration
+
+    /// Picker for setting the app's color scheme.
+    @ViewBuilder
+    var appColorSchemePicker: some View {
+        Text("App Color Scheme")
+            .font(.headline)
+        Picker("Color Scheme", selection: $configuration.colorScheme) {
+            Text("Light").tag(ColorScheme.light)
+            Text("Dark").tag(ColorScheme.dark)
+        }
+        .pickerStyle(.segmented)
+    }
+
+    /// Picker for setting the current Fluent theme.
+    @ViewBuilder
+    var themePicker: some View {
+        Text("Theme")
+            .font(.headline)
+        Picker("Theme", selection: $configuration.theme) {
+            Text("\(DemoColorTheme.default.name)").tag(DemoColorTheme.default)
+            Text("\(DemoColorTheme.green.name)").tag(DemoColorTheme.green)
+        }
+        .pickerStyle(.segmented)
+    }
+
+    /// Collects the various pickers and toggles together.
+    @ViewBuilder
+    var contents: some View {
+        VStack {
+            appColorSchemePicker
+            FluentDivider()
+                .padding()
+
+            themePicker
+            FluentDivider()
+                .padding()
+
+            Text("Control")
+                .font(.headline)
+
+            // Theme-wide override toggle
```

Comment:
It seems like GitHub won't let me reply to this [comment](https://github.com/microsoft/fluentui-apple/pull/879#discussion_r792178036):

I think it'd be nice to have an API in the design token system to tell whether the tokens are overridden for a specific control in a given theme. That way we could set the state of the toggle when the UI is being created.